apiserver/params/network.go

```diff
@@ -711,10 +711,10 @@ type NetworkInfo struct {
 // NetworkInfoResult Adds egress and ingress subnets and changes the serialized
 // `Info` key name in the yaml/json API protocol.
 type NetworkInfoResult struct {
-	Error            *Error        `json:"error,omitempty" yaml:"error,omitempty"`
-	Info             []NetworkInfo `json:"bind-addresses,omitempty" yaml:"bind-addresses,omitempty"`
-	EgressSubnets    []string      `json:"egress-subnets,omitempty" yaml:"egress-subnets,omitempty"`
-	IngressAddresses []string      `json:"ingress-addresses,omitempty" yaml:"ingress-addresses,omitempty"`
```

Comment:
This feels like it would be a version compatibility issue. You can test this by bootstrapping a controller, deploying something into the default model, then upgrading the controller, and seeing if there is an issue running 'network-get' on the model that hasn't been upgraded.